Directory.Build.targets

```diff
@@ -1,8 +1,8 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
     <EFCoreVersion>5.0.0-rc.1.20451.13</EFCoreVersion>
     <MicrosoftExtensionsVersion>5.0.0-rc.1.20451.14</MicrosoftExtensionsVersion>
-    <NpgsqlVersion>5.0.0-ci.20201003T171056</NpgsqlVersion>
+    <NpgsqlVersion>5.0.0-ci.20201017T090100</NpgsqlVersion>
```

Comment:
Needed to update to pull [these changes ](https://github.com/npgsql/npgsql/commit/2bdb08d38e8e139f9b466468c5383d2c6e8766ad)in.